types/tests.ts

```diff
@@ -25,30 +24,29 @@ setResolver(EmberResolver.create());
 module('rendering', function (hooks) {
   setupRenderingTest(hooks);
 
-  test('it renders', function (this: RenderingTestContext, assert) {
+  test('it renders', async function (assert) {
     assert.expect(2);
 
     // setup the outer context
     this.set('value', 'cat');
 
     // render the component
-    this.render(hbs`
+    await render(hbs`
             {{ x-foo value=value action="result" }}
         `);
 
     // has to be a template
-    // @ts-expect-error
-    this.render();
-    // @ts-expect-error
-    this.render('{{ x-foo value=value action="result" }}');
+    await render(hbs);
+    
+    await render(hbs`{{ x-foo value=value action="result" data-test-content}}`);
 
     const el = this.element.querySelector('div');
```

Comment:
@NullVoxPopuli here

Reply:
I would think it would be okay to take line 43 out and change line 44 to something like 

```js
assert.equal('[data-test-content]', 'cat', 'The component shows the correct value');
```